src/public/publicAPIs.ts

```diff
@@ -328,3 +329,17 @@ export function initializeWithFrameContext(
   initialize(callback, validMessageOrigins);
   setFrameContext(frameContext);
 }
+
+/**
+ * register a handler to be called when a user setting changes. The changed setting key & value is provided in the callback.
+ * @param settingKeys List of user setting changes to subscribe
+ * @param handler When a subscribed setting is updated this handler is called
+ */
+export function registerUserSettingsChangeHandler(
+  settingKeys: UserSettingKeys[],
```

Comment:
We just need to extend registerHandler to support args in this case settingKeys and use it here.